vscode_remote_tunnel/start_vscode_tunnel.py

```diff
@@ -145,10 +147,18 @@ def main(args):
         metavar="",
     )
 
+    PARSER.add_argument(
+        "-g",
+        "--gpu",
+        help="No. of GPUs to request",
+        default=0,
+        metavar="",
+    )
+
     PARSER.add_argument(
         "-m",
-        "--mem-per-cpu",
-        help="Memory per CPU to request",
+        "--mem",
+        help="Total memory to request",
         default="8G",
```

Comment:
Now that we are switching to `--mem`, please bump up the default to `16G` as the script requests 2 cpus by default. `12G` may be acceptable as well.

Reply:
It was always requesting the total memory. I only changed the wording so it reflects what it was requesting. If 8G was working fine before, we can just leave it as is?

Reply:
Good catch. Let's leave it as it is :)